### DISCUSSION TITLE
How do I clean up test data from the registry?

### DISCUSSION THREAD

**Discussion by ganeshbhat-tekdi** [Sunbird-RC/community#313] [General]
We are building a system in which User entities are created in the registry. A User signs up using a phone number and proceeds to complete eKYC via AADHAR. On successful eKYC, an AADHAR reference ID is stored for the user.
There are various test scenarios around this flow and a lot of test users are created during various tests, including load tests. 
Is there a way to completely remove all the test data from the registry, rather than doing soft deletes for each?

**Comment by tejash-jl** [comment]
@ganeshbhat-tekdi 
right now you need to manually purge the DB & ES.

**Reply by ganeshbhat-tekdi** [reply]
Thanks Tejash!
We are also looking for a solution to do a selective cleanup of test users, as other users are real. Test users may have to repeat the registration process in the next test cycle. Any thoughts on that would be most helpful.
